types/react-expand-animated/index.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for react-expand-animated 1.0
+// Project: https://github.com/sonybinhle/react-expand-animated
+// Definitions by: Ben Langlois <https://github.com/BenLanglois>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+import * as CSS from 'csstype';
+
+export interface ExpandProps {
+    /**
+     * The expanded or collapsed content
+     */
+    children: React.ReactNode;
+
+    /**
+     * When set to true expand the children component otherwise collapse it
+     */
+    open?: boolean;
+
+    /**
+     * Animation duration in ms
+     */
+    duration?: number;
+
+    /**
+     * Css3 Animation's type
+     */
+    easing?: CSS.Property.TransitionTimingFunction;
```

Comment:
I don't think there was a requirement here to use `csstypes` and the type should be a simple, scalar value. I assume author's intention was to use a string everywhere, say:

> easing	string	false	'ease-in-out'	Css3 Animation's type

or:

https://github.com/sonybinhle/react-expand-animated/blob/master/src/Expand.jsx#L67

(that's not possible to retain those other types when concatenating into string)

Adding `csstypes` also adds 3rd party dependency to library, so those large (20k+ LOC) will install when using those types.

Reply:
https://github.com/sonybinhle/react-expand-animated/blob/master/src/Expand.jsx#L137

Reply:
Thanks for looking into this. The library expects a transition timing function like `ease` or `ease-in` etc. in the code samples you link to, so I think this is a sensible type. Note that this type is effectively a string union type, so it does fit those code snippets.

On adding the dependency, given that React's types depend on `csstypes` too and anyone using this would being using React's types already I don't think this is a great concern.

However, I am also happy for this to be a simple string type, as I said in my previous review :)

Reply:
I can be missing something, the function I linked just consumes and expect `easing` to be a string, it's just that simple

Reply:
`CSS.Property.TransitionTimingFunction` is equivalent to `"ease" | "ease-in" | "ease-in-out" | "ease-out" | "step-end" | "step-start" | "linear" | string`

Yes, this is effectively just a string, but using this type more clearly indicates what is expected and improves tools like VS Code's intellisense as it will suggest these union type values (while still allowing arbitrary strings if the user desires - primarily to support custom easing functions).